### ISSUE TITLE
Put something web in tft screen

### ISSUE THREAD

**Issue by city8208** [vesteraas/node-pitft#16]
Hey,did anyway to put a web page on the screen?

I know how to put picture on screen and also text/Button.
But I didn't found a way to put website (like nodejs gcode viewer) on screen and show it.

**Comment by Turbine1991** [comment]
You might be best to point your framebuffer to a desktop environment.

Displaying a webpage is not easy to make simple.

**Comment by thebitguru** [comment]
I agree with @Turbine1991.  Checkout this page as one example: https://www.sodesne.com/blog/2019/4/15/setting-up-raspberry-pi-with-octoprint-and-pitft
